packages/monitors/src/SyntheticPegMonitor.js

```diff
@@ -289,17 +289,31 @@ class SyntheticPegMonitor {
     const latestTime = pricefeed.getLastUpdateTime();
     const volData = this._calculateHistoricalVolatility(pricefeed, latestTime, this.volatilityWindow);
     if (!volData) {
+      // If missing vol data, then the pricefeed is failing to return historical price data. Let's
+      // try catching the pricefeed's error to get more details about the problem:
+      let priceFeedErrorDetails;
+      try {
+        pricefeed.getHistoricalPrice(latestTime);
+      } catch (err) {
+        priceFeedErrorDetails = err.message;
+      }
       return {
         errorData: {
-          latestTime: latestTime ? latestTime : 0
+          latestTime: latestTime ? latestTime : 0,
+          priceFeedErrorDetails
```

Comment:
nit: thoughts on just getting _calculateHistoricalVolatility() to keep track of its last error and throwing it in the spot where it currently returns null?

Reply:
Hmm this is pretty interesting. So have _calculateHistoricalVolatility return successfully or throw, and have _checkPricrfeedVol report an error if it exists